### ISSUE TITLE
Taskbar progress still necessary?

### ISSUE THREAD

**Issue by ygoe** [codecadwallader/codemaid#761]
### Environment
- Visual Studio version: 2019 Community
- CodeMaid version: 11.1
- Code language: C#

### Description
Is the option "Show build progress on Visual Studio icon in Windows taskbar" in the "Progressing" category still relevant? It interferes with the Solution Icon extension in that it removes that overlay icon, and when I disable the option, I still see build progress on the taskbar icon. The difference seems to be that there's no indeterminate progress at build start anymore, but only the progress of completed projects. I can't verify that exactly.

That option seems to be active by default. Disabling it adds a new entry in the solution's CodeMaid.config file.

See also https://github.com/ashmind/SolutionIcon/issues/9

**Comment by codecadwallader** [comment]
Thanks for asking the question.  It looks like another difference is the native one doesn't show a red status when a build starts to fail.  Did turning off the option resolve the conflict you were seeing with another extension?

**Comment by ygoe** [comment]
Yes, with this option disabled the solution icon remained in place and never disappeared.

I remember I was happy about this progress indicator some 10 years ago when I worked with a solution of 25 projects. I can't remember though if it was CodeMaid or one of the Power Tools that I was using then, probably CodeMaid. I never really noticed that indicator with solutions of less projects.

**Comment by DamianSuess** [comment]
I like having the progress bar. Especially for our one C++ project which takes 15 minutes to build. It's a great addition, please keep the progress bar 👍

**Comment by ygoe** [comment]
@DamianSuess My question is not about removing the progress bar. It's about that Visual Studio can already do that on its own without CodeMaid. Can you please try out what happens when you disable the progress option in CodeMaid and start a rebuild of the solution. Do you still see the progress on the taskbar? My observation is with Visual Studio 2019. Which version do you have?
